### PR TITLE
Feature/e2e testing ci

### PR DESCRIPTION
**Pull request by StyleT**

### CODE REVIEW

File: e2e/package.json [hunk 1]
```diff
@@ -5,9 +5,10 @@
     "main": "index.js",
     "private": true,
     "scripts": {
-        "test:e2e": "cross-env TEST_ENV=verbose npm run test:e2e:ci -- --verbose",
-        "test:e2e:generate-typings": "codeceptjs def -o ./typings",
-        "test:e2e:ci": "cross-env NODE_ENV=test codeceptjs run"
+        "start": "codeceptjs run",
+        "start:verbose": "cross-env TEST_ENV=verbose npm start --- --verbose",
```
Comment (oleh-momot) [inline e2e/package.json:9]: Probably, it should be `--` instead of `---`

File: install_all.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ const commands = [
     { command: 'cd ./ilc/ && npm i', name: 'ilc' },
     { command: 'cd ./registry/ && npm i', name: 'registry' },
     { command: 'cd ./registry/client && npm i', name: 'registry:ui' },
+    { command: 'cd ./e2e && npm i', name: 'registry:ui' },
```
Comment (oleh-momot) [inline install_all.js:7]: I guess it should be `e2e` here instead of `registry:ui`

File: e2e/codecept.presettings.js [hunk 3]
```diff
@@ -2,75 +2,70 @@ const waitOn = require('wait-on');
...
+            childProcess.once('error', (error) => {
                 reject(error);
             });
+            childProcess.once('exit', () => !childProcess.killed && done(new Error('Child process exited...')));
 
             waitOn({
-                resources: ports.map((port) => `tcp:localhost:${port}`),
-                timeout: 60 * 1000,
+                resources,
+                timeout: 10 * 60 * 1000,
```
Comment (oleh-momot) [inline e2e/codecept.presettings.js:57]: Are you sure that we need to have such a timeout?

File: install_all.js (hunk 2 above)
Comment (StyleT) [reply install_all.js]: done, thx

File: e2e/package.json (hunk 1 above)
Comment (StyleT) [reply e2e/package.json]: done, thx

File: e2e/codecept.presettings.js (hunk 3 above)
Comment (StyleT) [reply e2e/codecept.presettings.js:57]: Yeah, at CI we're pulling Docker image with demo apps so it may take a couple of minutes...

**Pull request merged**